apm-sniffer/apm-sdk-plugin/pulsar-common/src/main/java/org/apache/skywalking/apm/plugin/pulsar/common/PulsarPluginConfig.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *   Licensed to the Apache Software Foundation (ASF) under one or more
+ *   contributor license agreements.  See the NOTICE file distributed with
+ *   this work for additional information regarding copyright ownership.
+ *   The ASF licenses this file to You under the Apache License, Version 2.0
+ *   (the "License"); you may not use this file except in compliance with
+ *   the License.  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.apache.skywalking.apm.plugin.pulsar.common;
+
+import org.apache.skywalking.apm.agent.core.boot.PluginConfig;
+
+public class PulsarPluginConfig {
+
+    public static class Plugin {
+        @PluginConfig(root = PulsarPluginConfig.class)
+        public static class Pulsar {
+            /**
+             * If set to true, the message contents of the Pulsar topic would be collected.
+             */
+            public static boolean TRACE_MESSAGE_CONTENTS = true;
```

Comment:
The default value should be false. The impact is clear for memory, network and database.

Reply:
agent.config set to false is not enough.

Reply:
Okay, I'll make some modifications and conduct unit testing. Thank you!